### PR TITLE
Update README_CN.md

### PR DESCRIPTION
**Pull request by linlinnj**
修改名称dev-manua-cn.md

NOTE: Please read the Contributing.md guidelines before submitting your patch:

https://github.com/apache/incubator-apisix/blob/master/Contributing.md#how-to-add-a-new-feature-or-change-an-existing-one

### Summary

SUMMARY_HERE

### Full changelog

* [Implement ...]
* [Add related tests]
* ...

### Issues resolved

Fix #XXX

### CODE REVIEW

File: doc/README_CN.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ Reference document
 * [APISIX 说明](../README_CN.md)
 * [架构设计](architecture-design-cn.md)
 * [压力测试](benchmark-cn.md)
-* [搭建开发环境](dev-manua-cnl.md)
+* [搭建开发环境](dev-manua-cn.md)
```
Comment (moonming) [inline doc/README_CN.md]: `dev-manua-cn` is also a 404 link, please check again.

Comment (membphis) [reply doc/README_CN.md]: `dev-manual-cn.md` correct address

Comment (juzhiyuan) [reply doc/README_CN.md]: It should be `dev-manual-cn.md` in `d714b914542b9a2e904cd5f76e611adb57935eb7` branch.

Comment (juzhiyuan) [reply doc/README_CN.md]: Just take a look at

`https://github.com/apache/incubator-apisix/blob/d714b914542b9a2e904cd5f76e611adb57935eb7/doc/README_CN.md`

and

`https://github.com/apache/incubator-apisix/blob/d714b914542b9a2e904cd5f76e611adb57935eb7/doc/`

**Comment by moonming** [comment]
@linlinnj any update?

**Comment by moonming** [comment]
@linlinnj any update?

**Pull request merged**